### PR TITLE
Things

### PR DESCRIPTION
**Pull request by matt-riemer**
Sorry I didn't stick to Issues or anything.

But this is a few things that bugged me!

### CODE REVIEW

File: lib/health/stats/log.ex [hunk 1]
```diff
@@ -16,6 +16,6 @@ defmodule Health.Stats.Log do
     log
     |> cast(attrs, [:weight, :date, :user_id])
     |> validate_required([:weight, :date, :user_id])
-    |> validate_number(:weight, greater_than: 100, less_than: 500)
+    |> validate_number(:weight, greater_than: 1, less_than: 500)
```
Comment (matt-riemer) [inline lib/health/stats/log.ex]: 100? really?

File: lib/health_web/pow/messages.ex [hunk 2]
```diff
@@ -0,0 +1,19 @@
+defmodule HealthWeb.Pow.Messages do
+  use Pow.Phoenix.Messages
+  use Pow.Extension.Phoenix.Messages, extensions: [PowResetPassword]
+
+  import HealthWeb.Gettext
+
+  # For a list of messages:
+  # https://github.com/danschultzer/pow/blob/master/lib/pow/phoenix/messages.ex
+
```
Comment (matt-riemer) [inline lib/health_web/pow/messages.ex]: These 3 messages are `nil` by default, but I felt they were missing. I wanted to play with pow.

File: lib/health_web/templates/log/index.html.eex [hunk 3]
```diff
@@ -1,14 +1,18 @@
 <div class="row">
   <div class="col-lg-6">
```
Comment (matt-riemer) [inline lib/health_web/templates/log/index.html.eex]: sorry this is a bad merge maybe

File: lib/health/stats/log.ex (hunk 1 above)
Comment (alanvardy) [reply lib/health/stats/log.ex]: I’m worried about anorexics using this code! And midgets!

If someone weighs 1 lb the have no business using this app!

File: lib/health_web/pow/messages.ex (hunk 2 above)
Comment (alanvardy) [reply lib/health_web/pow/messages.ex]: Lol... I added my own too before... we may have a conflict

File: lib/health_web/views/layout_view.ex [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
+  # Change alert-error to alert-danger
+  def bootstrap_flash(conn) do
+    Enum.map(get_flash(conn), fn { key, message} ->
+      case key do
+        "error" -> { "danger", message }
+        _ -> { key, message }
+      end
+    end
+   )
+  end
```
Comment (alanvardy) [inline lib/health_web/views/layout_view.ex]: This is cool!

File: lib/health_web/pow/messages.ex (hunk 2 above)
Comment (matt-riemer) [reply lib/health_web/pow/messages.ex]: Yea i just discovered this. Don't worry I deleted yours :)

**Comment by alanvardy** [comment]
Cool! Thanks for the PR.. I will finish reviewing in the morning

File: lib/health_web/pow/messages.ex (hunk 2 above)
Comment (alanvardy) [reply lib/health_web/pow/messages.ex]: Haha

**Comment by alanvardy** [comment]
If you click on the circleci details it will be happy to inform you that your line lengths are still too damn long :)

**Comment by matt-riemer** [comment]
Do you even `mix credo --strict` bro?

**Comment by matt-riemer** [comment]
Funny thing about that line length

The line was IN A COMMENT. and it wasn't even my code, it was a straight up copy and paste from Pow.

**Comment by alanvardy** [comment]
Lol... I love some strict credo. Thanks for the PR!

I just made a PR on bodyguard yesterday because their example code brought up a deprecation warning.

I’m sure that you just looked at that line length, shrugged and said, ‘eh!’ :)

**Pull request merged**